### ISSUE TITLE
Enhancement for install.sh on website

### ISSUE THREAD

**Issue by Znuff** [litespeedtech/cyberpanel#273]
I would have made a pull request, but I can't seem to find this file in the repository.

```
#!/bin/bash

if [[ -f "/etc/os-release" ]]; then 
  source /etc/os-release
else
  ID="unsupported"
  PRETTY_NAME="Your OS does not have a /etc/os-release file"
fi

if [[ "$ID" == "ubuntu" && "$UBUNTU_CODENAME" == "bionic" ]]; then
  export DEBIAN_FRONTEND=noninteractive
  apt -q -y -o Dpkg::Options::=--force-confnew update
  apt -q -y -o Dpkg::Options::=--force-confnew install wget curl
elif [[ ( "$ID" == "centos" || "$ID" == "cloudlinux" ) && ( "$VERSION_ID" == 7 || "$VERSION_ID" == 7.* ) ]]; then
  yum install curl wget -y 1> /dev/null
  yum update curl wget ca-certificates -y 1> /dev/null
  SERVER_OS="$NAME"
elif [[ ( "$ID" == "centos" || "$ID" == "cloudlinux" ) && ( "$VERSION_ID" == 8 || "$VERSION_ID" == 8.* ) ]]; then
  echo "CentOS 8/CloudLinux 8 support is currently experimental!"
  yum install curl wget -y 1> /dev/null
  yum update curl wget ca-certificates -y 1> /dev/null
  SERVER_OS="${NAME}${VERSION_ID}"
else 
  echo "Your OS -- ${PRETTY_NAME} -- is not currently supported!"
  echo "CyberPanel is currently supported on Ubuntu 18.04, CentOS 7 and CloudLinux 7."
  exit 1
fi

rm -f cyberpanel.sh install.tar.gz 
curl --silent -o cyberpanel.sh "https://cyberpanel.sh/?dl&${SERVER_OS}" 2>/dev/null
chmod +x cyberpanel.sh
./cyberpanel.sh "$@"
```

This does the following:

- Relies on /etc/os-release file for os-detection, all modern OS that you are currently targetting (CentOS 7+, Ubuntu 18+ etc.) support this -- for more information about this file, please check http://man7.org/linux/man-pages/man5/os-release.5.html
- It also retains `SERVER_OS` as a variable, even tough I don't really see why you are using it, other than for statistics gathering purposes server side -- I have double-checked the output of the `cyberpanel.sh` script and it seems that the output is identical no matter what input it gets -- I honestly recommend removing it all-together if it's not used, and instead rely on something else to pass to the download.

**Comment by qtwrk** [comment]
Hi,

Thanks for the suggestion.

Yes,  the install.sh was kind of "legacy" remanent long time ago , before start to use github , so didn't put the file in github as kind of feel that script won't be edited for any foreseeable future.

I just put the that file into github , please note that all other shell script are "bash" , but this one is actually "sh".

And yes, it was for statistics.

Best regards,

**Comment by Znuff** [comment]
I am aware that it was `sh` instead of `bash`, I'm just wondering if it makes any sense to have it that way, since further along the way you are actually relying on `bash` functionality anyway.

I can rewrite it to be POSIX-Compatible.

**Comment by qtwrk** [comment]
> I am aware that it was `sh` instead of `bash`, I'm just wondering if it makes any sense to have it that way, since further along the way you are actually relying on `bash` functionality anyway.
> 
> I can rewrite it to be POSIX-Compatible.

Yes, that's another legacy issue ,   originally it was using the way like `sh <(curl .........)` , later on we changed it to bash script , but people are still using `sh`  command to execute it , so I had make that sh script to call the bash script to avoid syntax problem between sh and bash.

**Comment by Znuff** [comment]
Made a pull request #274

**Comment by usmannasir** [comment]
Pull request merged, and also merged to v1.9.5 branch. Your changes will come when this version is released.

thanks.

**Comment by qtwrk** [comment]
I made some changes to remove white space and unwanted string to keep to compatible with current statistics code.
